core/serializers.py

```diff
@@ -109,20 +109,21 @@ class CoreUserWritableSerializer(CoreUserSerializer):
     Override default CoreUser serializer for writable actions (create, update, partial_update)
     """
     password = serializers.CharField(write_only=True)
+    organization_name = serializers.CharField(source='organization.name')
     core_groups = serializers.PrimaryKeyRelatedField(many=True, queryset=CoreGroup.objects.all(), required=False)
 
     class Meta:
         model = CoreUser
-        fields = CoreUserSerializer.Meta.fields + ('password',)
+        fields = CoreUserSerializer.Meta.fields + ('password', 'organization_name')
         read_only_fields = CoreUserSerializer.Meta.read_only_fields
 
     def create(self, validated_data):
         # get or create organization
         try:
             organization = validated_data.pop('organization')
         except (KeyError):
-            organization = settings.DEFAULT_ORG
-        organization, is_new_org = Organization.objects.get_or_create(name=organization)
+            organization = {'name': settings.DEFAULT_ORG}
```

Comment:
Are we adding organization_name and keeping organization for backwards comparability?  Not sure this reconciles with the fixture test if true.

Reply:
Yes, it is for backward compatibility and this aligns with the fixture as that is changed to organization_name alone